substrate/frame/llm/src/lib.rs

```diff
@@ -502,8 +502,8 @@ pub mod pallet {
 
 		/// Emit Remarked event. Used by Liberland tooling to annotate transfers.
 		#[pallet::call_index(7)]
-		#[pallet::weight(<T as Config>::WeightInfo::treasury_lld_transfer())]
-		pub fn remark(origin: OriginFor<T>, data: RemarkData) -> DispatchResult {
+		#[pallet::weight(<T as Config>::WeightInfo::remark(data.len() as u32))]
+		pub fn remark(_origin: OriginFor<T>, data: RemarkData) -> DispatchResult {
```

Comment:
`origin` is unused, so compiler was throwing warnings about it. `_` marks it as unused and fixes the warning.